src/applications/hca/reducer.js

```diff
@@ -67,6 +77,34 @@ export function hcaEnrollmentStatus(state = initialState, action) {
     case SHOW_HCA_REAPPLY_CONTENT:
       return { ...state, showHCAReapplyContent: true };
 
+    case FETCH_DISMISSED_HCA_NOTIFICATION_STARTED:
+      return { ...state, isLoadingDismissedNotification: true };
+
+    case FETCH_DISMISSED_HCA_NOTIFICATION_SUCCEEDED: {
+      const {
+        statusEffectiveAt: dismissedNotificationDate,
+      } = action.data.data.attributes;
```

Comment:
`action.payload.data.attributes` or `action.response.data.attributes` might be better

Reply:
duh :-/